lib/Sema/CSGen.cpp

```diff
@@ -1168,7 +1166,7 @@ namespace {
         CS.addBindOverloadConstraint(fnTy, choice, subscriptMemberLocator,
                                      CurDC);
       } else {
-        CS.addValueMemberConstraint(baseTy, Context.Id_subscript,
+        CS.addValueMemberConstraint(baseTy, DeclName::createSubscript(),
```

Comment:
It's the difference between a reference to "any subscript" and "a subscript with zero arguments", or between a base DeclName and a compound DeclName. Just wanted to make sure this was producing a base DeclName and not a compound one.